### ISSUE TITLE
Option for add @Expose annotation

### ISSUE THREAD

**Issue by kasim1011** [wuseal/JsonToKotlinClass#73]
Please add option for adding `@Expose` annotation to all values.

**Comment by wuseal** [comment]
@kasim1011 Thanks for your suggestion, Maybe any reason?

**Comment by kasim1011** [comment]
yes, `Gson` configuration. `GsonBuilder().excludeFieldsWithoutExposeAnnotation().create()`.

**Comment by wuseal** [comment]
@kasim1011 Now, By custom your own annotation could achieve your goal.
The option in the annotation tab: `Others by customize`
